Packs/VaronisSaaS/Integrations/VaronisSaaS/VaronisSaaS.yml

```diff
@@ -0,0 +1,556 @@
+category: Analytics & SIEM
+commonfields:
+  id: VaronisSaaS
+  version: -1
+configuration:
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+  defaultvalue: Varonis SaaS Incident
+- display: The FQDN/IP the integration should connect to
+  hidden: false
+  name: url
+  required: true
+  type: 0
+- display: X-API-Key
+  hidden: false
+  name: apiKey
+  required: true
+  type: 4
```

Comment:
Sorry, I missed the "hiddenusername: true"
however the question is still the same: why we should make it as "Password" instead of "X-API-Key"?
Here is how updated screen looks:

![image](https://github.com/demisto/content/assets/145133825/0be51f1a-b8e2-438a-9d45-bb660513d105)

Reply:
https://github.com/demisto/content/pull/31989#issuecomment-1898807351